tflite2onnx/op/binary.py

```diff
@@ -99,10 +102,11 @@ def parse(self):
 
         # options
         op_opt = op.BuiltinOptions()
-        option = self.OptionMapping[opcode]()
-        option.Init(op_opt.Bytes, op_opt.Pos)
+        if opcode in self.OptionMapping:
+            option = self.OptionMapping[opcode]()
+            option.Init(op_opt.Bytes, op_opt.Pos)
 
-        handleFusedActivation(self, option, ot)
+            handleFusedActivation(self, option, ot)
```

Comment:
Thanks for fixing this, I don't even know there is an issue here.

Reply:
Power is a binary operation, but it doesn't have any options. Therefore, I added this check.